student/src/main/java/tds/student/web/handlers/MasterShellHandler.java

```diff
@@ -141,6 +137,10 @@
   @Autowired
   private ItemBankService         itemBankService;
 
+  @Autowired
+  @Qualifier("integrationOpportunityService")
```

Comment:
This is our remote opportunity service.  It still calls the legacy one in addition to making the http call.